import-export-cli/cmd/addApi.go

```diff
@@ -196,25 +204,63 @@ func createAPI(name string, namespace string, configMapName string, replicas int
 	}
 }
 
-func handleInterceptors(configMapName string, path string, operation string, namespace string) {
+func handleBalInterceptors(configMapName string, path string, operation string, namespace string) string {
 
 	//get interceptors if available
 	interceptorsPath := filepath.Join(path, filepath.FromSlash("Interceptors"))
 	//check interceptors dir is not empty
 	file, err := os.Open(interceptorsPath)
 	if err != nil {
 		utils.HandleErrorAndExit("cannot open interceptors Dir", err)
+		return ""
 	}
 	defer file.Close()
 	_, err = file.Readdir(1)
 	if err == nil {
 		//creating kubernetes configmap with interceptors
-		fmt.Println("creating configmap with interceptors")
+		fmt.Println("creating configmap with ballerina interceptors")
 		errConfInt := createConfigMapWithNamespace(configMapName, interceptorsPath, namespace, operation)
 		if errConfInt != nil {
 			utils.HandleErrorAndExit("Error creating configmap for interceptors", err)
 		}
+		return configMapName
+	}
+	return ""
+}
+
+func handleJavaInterceptors(path string, operation string, namespace string, flagApiName string) []string {
+	var interceptors []string
+	var javaInterceptorsConfNames []string
+	//get interceptors if available
+	interceptorsPath := filepath.Join(path, filepath.FromSlash("libs"))
+	//check interceptors dir is not empty
+	file, err := os.Open(interceptorsPath)
+	if err != nil {
+		utils.HandleErrorAndExit("cannot open interceptors Dir", err)
```

Comment:
You've added a `return ""` in `handleBallerinaInterceptors` method. What about here? not required?

Reply:
Yes not required. Since we have multiple jar files when adding java interceptors we are adding each in different configmaps. Therefore we are returning a string array which contains the names of those jar files.